addons/mail/static/tests/discuss/call/call_settings_menu.test.js

```diff
@@ -50,11 +60,16 @@ test("Renders the call settings", async () => {
     await contains("label[aria-label='Camera']");
     await contains("label[aria-label='Microphone']");
     await contains("label[aria-label='Speakers']");
-    await contains("option", { textContent: "Permission Needed", count: 3 });
+    await contains(".o-mail-DeviceSelect-button", { textContent: "Permission Needed", count: 3 });
     rtc.microphonePermission = "granted";
-    await contains("option[value=mockAudioDeviceId]");
+    const browserDefaultLabel = isBrowserChrome() ? "Default" : "Browser Default";
+    await click(".o-mail-DeviceSelect-button[data-kind='audioinput']", { textContent: "Default" });
+    await contains(".o-dropdown-item", { textContent: "mockAudioDeviceLabel" });
+    await contains(".o-dropdown-item", { textContent: browserDefaultLabel });
     rtc.cameraPermission = "granted";
-    await contains("option[value=mockVideoDeviceId]");
+    await click(".o-mail-DeviceSelect-button[data-kind='videoinput']", { textContent: "Default" });
+    await contains(".o-dropdown-item", { textContent: "mockVideoDeviceLabel" });
+    await contains(".o-dropdown-item", { textContent: browserDefaultLabel });
```

Comment:
Can use `:has(:text())` to target `textContent` on current node and any siblings. This is the preferred syntax over `text` and `textContent` params of `contains()`.